faker/providers/phone_number/pt_PT/__init__.py

```diff
@@ -19,6 +20,7 @@ class Provider(PhoneNumberProvider):
         '(351) 93# ### ###',
         '(351) 96# ### ###',
         '(351) 2## ### ###',
+        '(351) 2# ### ### #)',
```

Comment:
If I'm reading https://en.wikipedia.org/wiki/Telephone_numbers_in_Portugal correctly, it's always `21`, ie:  there isn't any number with a prefix if `22`, `23`, etc. Can you confirm?

Reply:
this is not truth, we have many prefix's, for example 21 is used to the Lisbon, 22 is to Porto, 252 is to Vila Nova de Famalicão and so one so forth